HW_3/Task_3.py

```diff
@@ -8,3 +8,23 @@
 # Будем считать, что на вход подается только одно слово, которое содержит либо только английские,
 # либо только русские буквы.
 
+eng = {1:'AEIOULNSTR',
+      	2:'DG',
+      	3:'BCMP',
+      	4:'FHVWY',
+      	5:'K',
+      	8:'JZ',
+      	10:'QZ'}
+rus = {1:'АВЕИНОРСТ',
+      	2:'ДКЛМПУ',
+      	3:'БГЁЬЯ',
+      	4:'ЙЫ',
+      	5:'ЖЗХЦЧ',
+      	8:'ШЭЮ',
+      	10:'ФЩЪ'}
+N = abs(int(input("Введите 0 для игры на русском языке, либо 1 для игры на английском языке: ")))
+word = input("Введите ваше слово: ").upper()
+if N == 1:
+	print(sum([k for i in word for k, v in eng.items() if i in v]))
```

Comment:
Функция сумм принимает любой итерируемый объект, а значит, можно передать туда прямой генератор.
```suggestion
	print(sum(k for i in word for k, v in eng.items() if i in v))
```